databases/week2/hyf_lesson02queries.sql

```diff
@@ -0,0 +1,42 @@
+USE hyf_lesson02;
+
+SELECT *
+FROM task;
+
+insert into task (id, title, description, created, updated, due_date, status_id) values (36, 'Complete the Assignment', 'work on your recent hyf assignment', '2022-02-06 06:54:16', '2022-02-07 13:05:09', '2022-02-11 13:05:09' , 2);
```

Comment:
you don't need to insert the `id`, and it's actually better not to insert it and let mysql auto increment it.